modules/default.nix

```diff
@@ -1,5 +1,12 @@
-{
+{lib, ...}: {
     imports = [
         ./stregsystemet.nix
     ];
+
+    options = {
+        domain = lib.mkOption {
```

Comment:
Personally i think this is a bad place to put this option, but i don't know where would be a better place to put it

Another solution might be to make a `services.stregsystem.domain` option, `fikien.domain` and whatever else we define, then define in a let clause in `system/configuration.nix` a `domain` variable to set in all domain variables later on, but i think that's a ton of unneccessary code... i think we could implement this in a smarter way

Reply:
Agreed. It might be best to just have an `fklub` namespace where all our custom options that don't belong anywhere else go.